repo/packages/S/scylladb/0/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Could you add an scm, similar to https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/A/arangodb3/9/package.json#L6